protocol/v2/blockchain/beacon/network.go

```diff
@@ -67,3 +67,21 @@ func (n Network) IsFirstSlotOfEpoch(slot phase0.Slot) bool {
 func (n Network) GetEpochFirstSlot(epoch phase0.Epoch) phase0.Slot {
 	return phase0.Slot(epoch * 32)
 }
+
+// EstimatedSyncCommitteePeriodAtEpoch estimates the current sync committee period at the given Epoch
+func (n Network) EstimatedSyncCommitteePeriodAtEpoch(epoch phase0.Epoch) uint64 {
+	return uint64(epoch) / 256 // EpochsPerSyncCommitteePeriod
```

Comment:
lets use a const instead of hard coded

Reply:
noted!
we have a dedicated task for this